.github/workflows/package.yml

```diff
@@ -0,0 +1,34 @@
+name: package
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: install make deps
+      run: |
+        sudo apt install texlive-latex-extra
```

Comment:
What do you mean with that?

Reply:
To try and make this less surprising, I was hoping we could pin the version of texlive-latex-extra that's being installed

ref: https://askubuntu.com/questions/92019/how-to-install-specific-ubuntu-deb-packages-with-exact-version

Reply:
Ok I never did something like that. Won't we run into problems once that specific version of the package isn't in the repos anymore? (I'm not using ubuntu as my daily driver so I don't know for sure if that happens).

But the only two components we rely on are `pdflatex` and `l3build`. I think it's very unlikely that they will change the "api".

Reply:
> But the only two components we rely on are pdflatex and l3build. I think it's very unlikely that they will change the "api".

Sorry if this is a basic question but is it not possible to install these separately? My experience with installing latex packages like this is it can take a lot of time since it's installing so many addons/packages.

Reply:
Oh yes that's true. Latex installation always takes quite some time. I'll check if the components are in a smaller package (not the `-extra` one).
When I created the workflow it was also kinda the thought, well if the workflow runs for half a minute longer or not :shrug: 

Actually the workflows already run in [my repo/fork](https://github.com/atticus-sullivan/catppuccinLatex/actions) so at the moment the whole workflow runs in ~1 minute (and it's true, the installation takes up most of the time).

I have a workflow for another repo where I created a custom docker image which has latex already installed, but loading/fetching that one also takes ~ 30 seconds (but I'm really no docker expert so one might be able to squeeze some performance there).

Reply:
> So what do you think? Is is important to get the action execute faster than it is right now?

Thanks for some investigation into it. It's certainly not something to block the merge on but figured I'd point it out anyways that installing a full tex live distribution in CI/CD is probably something we'd like to avoid at some point in the future!

Reply:
Ok to be fair, there are github actions you can use to compile latex stuff, but that again doesn't work with l3build (I think) and it's a pain to do customization.